person-database/src/main/java/io/pravega/example/persondatabase/PersonDatabase.java

```
@@ -0,0 +1,109 @@
+        return config;
+    }
+
+
+    public void run() {
+        if (PravegaAppConfiguration.isCreateScope()) {
+            try (StreamManager streamManager = StreamManager.create(getConfig().getClientConfig())) {
+                streamManager.createScope(getConfig().getDefaultScope());
+            }
+        }
+
+        // Create Pravega stream.
+        PravegaUtil.createStream(getConfig().getClientConfig(), getConfig().getOutputStreamConfig());
+
+        try (EventStreamClientFactory clientFactory = EventStreamClientFactory.withScope(
+                getConfig().getOutputStreamConfig().getStream().getScope(),
+                getConfig().getClientConfig());
+             EventStreamWriter<ByteBuffer> pravegaWriter = clientFactory.createEventWriter(
+                     getConfig().getOutputStreamConfig().getStream().getStreamName(),
+                     new ByteBufferSerializer(),
+                     EventWriterConfig.builder().build())) {
+
+            ObjectMapper mapper = new ObjectMapper();
+
+            // args
+            String personId = getConfig().getpersonId();
+            String transactionType = getConfig().gettransactionType();
+            String imageName = "";
+
+            Date date = new Date();
+            long time = date.getTime();
+            Timestamp timestamp = new Timestamp(time);
+
+            // Decode the image.
+            BufferedImage originalImage = null;
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+            byte[] imageData = null;
+            if (transactionType.equals("add")) {
+                try {
+                    File imageFile = new File(getConfig().getimagePath());
+                    imageName = imageFile.getName();
+                    originalImage = ImageIO.read(imageFile);
+                    ImageIO.write(originalImage, "jpg", baos);
+                } catch (IOException e) {
+                    e.printStackTrace();
```

You should never just ignore errors like this. This makes it impossible to detect problems in scripts. Remove the try/catch completely. The outer-level try/catch will handle throw a RuntimeException.